python/ray/actor.py

```diff
@@ -1336,6 +1378,8 @@ def _actor_method_call(
             max_retries: Number of retries when method fails.
             retry_exceptions: Boolean of whether you want to retry all user-raised
                 exceptions, or a list of allowlist exceptions to retry.
+            task_tracing: True if tracing is enabled, i.e. task events from
```

Comment:
is this name the one that's agreed by everyone? 

1. Feel like the word "tracing" is confusing here since it overlaps with our tracing feature.
2. Generally it is more natural to use verb_noun style name?

Reply:
updated naming (to `enable_task_events`)